java/core/src/java/org/apache/orc/CollectionColumnStatistics.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.orc;
+
+/**
+ * Statistics for all of collections such as Map and List.
+ */
+public interface CollectionColumnStatistics extends ColumnStatistics {
+  /**
+   * Get minimum number of children in the collection.
+   * @return the minimum children count
+   */
+  long getMinimumChildren();
+
+  /**
+   * Get maximum number of children in the collection.
+   * @return the maximum children count
+   */
+  long getMaximumChildren();
+
+  /**
+   * Is the total defined? If the sum overflowed the counter this will be false.
+   * @return is the total no. of children available
+   */
+  boolean isTotalDefined();
```

Comment:
We don't need isTotalDefined. It isn't possible to overflow the long behind it.

Reply:
Sure, will take it out.